src/main/flight/pid.c

```diff
@@ -1068,26 +1122,53 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 
         // -----calculate I component
         float Ki = pidRuntime.pidCoefficient[axis].Ki;
+        float itermLimit = pidRuntime.itermLimit; // windup fraction of pidSumLimit
+
 #ifdef USE_LAUNCH_CONTROL
         // if launch control is active override the iterm gains and apply iterm windup protection to all axes
         if (launchControlActive) {
             Ki = pidRuntime.launchControlKi;
         } else
 #endif
         {
+
+            // handle yaw iTerm limit differently from other axes, and make yaw iTerm leaky
             if (axis == FD_YAW) {
+                itermLimit = pidRuntime.itermLimitYaw; // windup fraction of pidSumLimitYaw
```

Comment:
why is  pidRuntime.itermLimit used for launchControlActive yaw?

Reply:
From what I can see, when Launch Control is active, there is code around 1190 that applies specific limits to iTerm that are tighter than the limits normally used in flight.  
I suppose those limits could have been applied here, but they are in a separate code block further down.
Not sure that I'm answering the question but didn't want to mess with Launch Control.